webapp/src/components/Modal/BidConfirmationModal/BidConfirmationModal.js

```diff
@@ -48,14 +111,35 @@ export default class BidConfirmationModal extends React.PureComponent {
             values={{
               parcels: parcels.length,
               token,
-              price
+              price: formatAmount(price)
             }}
           />
         </div>
 
+        {!isLoading && balance != null ? (
+          <React.Fragment>
+            <div className="your-balance">
+              {t('auction_modal.your_balance', {
+                balance: formatAmount(balance),
+                token
+              })}
+            </div>
+            {balance < price ? (
```

Comment:
Hmmm I don't know if we are already doing it, but I think we should change the price with the margin (fee applied)

Reply:
Wouldn't it be weird that we show one price before hitting purchase and a different one on the confirmation modal? I we do so, I think we should show the price with margin as the big price next to the purchase button

Reply:
I will do it cause if we let the user proceed with less than the amount he should have, it will fail.

Is it ok to do this in this PR? @cazala @NicoSantangelo